mtags/src/main/scala/scala/meta/internal/metals/ScaladocIndexer.scala

```diff
@@ -11,7 +11,6 @@ import scala.meta.internal.semanticdb.Scala.Descriptor
 import scala.meta.internal.semanticdb.Scala.Symbols
 import scala.meta.internal.semanticdb.SymbolInformation
 import scala.meta.internal.semanticdb.SymbolOccurrence
-import scala.meta.internal.trees.Origin
```

Comment:
We could also do:
```
import scala.meta.internal.trees._
import scala.meta.trees._
```
which should avoid the need for the compat sections maybe?
Just came up with that, sorry :/

Reply:
In old interface we need to use `parsed.pos.start` and in new `parsed.begTokenIdx`, can we still avoid compat?

Reply:
Ok, then we need to keep it as is, sorry!